### PR TITLE
go-fuzz-build: added support for '-tags'

### PR DESCRIPTION
**Pull request by obscuren**
Please let me know if this is something that might be of interest and/or if you want me to expand further.

### CODE REVIEW

File: go-fuzz-build/main.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 )
 
 var (
+	flagTag  = flag.String("tags", "", "a list of build tags to consider satisfied during the build.")
```
Comment (Kubuxu) [inline go-fuzz-build/main.go]: Proper way of doing it is using: https://godoc.org/golang.org/x/tools/go/buildutil#TagsFlag

File: go-fuzz-build/main.go [hunk 2]
```diff
@@ -32,6 +33,14 @@ var (
 	GOROOT  string
 )
 
+func makeTags() string {
+	tags := "gofuzz"
+	if len(*flagTag) > 0 {
+		tags += " " + *flagTag
+	}
+	return tags
```
Comment (Kubuxu) [inline go-fuzz-build/main.go]: Joined tags should have form `-tags="tagA,tagB"` try it out for yourself.

See also: https://github.com/golang/go/issues/8772#issuecomment-253946071

Thins get a bit tricky when you have multiple tags in play.

Comment (obscuren) [inline go-fuzz-build/main.go]: Yes, indeed. This works though, it should be a space separated list of tags.

File: go-fuzz-build/main.go (hunk 1 above)
Comment (obscuren) [inline go-fuzz-build/main.go]: Thanks! I was not aware there was. Tags flag

File: go-fuzz-build/main.go (hunk 2 above)
Comment (obscuren) [inline go-fuzz-build/main.go]: If you mean that they need to be quoted, that's handled by the `exec.Command`

File: go-fuzz-build/main.go (hunk 1 above)
Comment (obscuren) [inline go-fuzz-build/main.go]: Now that I'm looking at it I'm not sure I want to pull in a new package just for a flag. I'll let @dvyukov decide on whether he'd like this or not.

Comment (dvyukov) [inline go-fuzz-build/main.go]: s/a list/a space-separated list/ please
also no dot at the end, other flags don't use dots

**Comment by dvyukov** [comment]
Otherwise looks good.

**Comment by obscuren** [comment]
Done

**Comment by dvyukov** [comment]
Thanks!

File: go-fuzz-build/main.go (hunk 2 above)
Comment (Kubuxu) [inline go-fuzz-build/main.go]: Ahh, rigth, sorry.

**Pull request merged**